ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -298,15 +298,19 @@ HWSRSR:
   - RSR
 
 ADY:
-  id: "15252673377812888960"
   description: "Air dryer."
   is_abstract: true
   implements:
   - EQUIPMENT
 
 CMP:
-  id: "15552673377812888960"
   description: "Air compressor."
   is_abstract: true
   implements:
+  - EQUIPMENT
+
+CRAC:
```

Comment:
Remove. Not functionally different from FCU, which is already defined.

Reply:
CRAC was removed